File: src/main/java/it/contrader/dto/ServerDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class ServerDTO {
	
		
		private Long serverId;
		
		private String posizione;
		
		private Long fk_id_url; 
		
		private String tipologia;
		
		private String data;
}

File: src/dto/serverdto.ts
/**
 * Classe DTO di Server. DEVE essere uguale (stesso nome classe, stessi attributi e stessi nomi) a
 * quello nel backend. 
 * 
 */
export class ServerDTO {

    id: number;
 
    posizione: string;
 
    tipologia: string;
 
    data: string;

    fkIdUrl: number;
 
 }